config/routes.rb

```diff
@@ -1,6 +1,40 @@
 Rails.application.routes.draw do
+<<<<<<< HEAD
+  get 'pages/contact'
+
+  get 'pages/about'
+
+  resources :musicians do
+    collection do
+      get 'top', to:"musicians#top"
+    end
+  end
+
+  resources :musicians do
+    member do
+      get 'instrument', to:"musicians#instrument"
+    end
+  end
+
+  resources :musicians do
+    resources :reviews, only:[:new, :create]
+  end
+end
+
+  namespace :admin do
+    resources :musicians, only: [:index]
+  end
+
+
+=======
   devise_for :users
   devise_for :admin_users
+
+  resources :clients
+  resources :bands
+  resources :musicians
+  resources :events
+>>>>>>> develop
   # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
This routes file needs to be revised and fixed. If deployed, this would totally break down the app.